docs/manual/templates.rst

```diff
@@ -155,6 +155,26 @@ Expressie                           Voorbeeld waarde                   Toelichti
 .. note:: Op dit moment krijgt u altijd de Nederlandse vertalingen/lokalisatie.
    Er is nog geen ondersteuning voor andere talen.
 
+Template tags
+-------------
+
+De volgende template tags kunnen ook worden gebruikt in opmaakcomponenten.
+
+**get_value**
+
+Geeft de mogelijkheid om een waarde uit een sleutel-waarde variabelen uit te halen.
```

Comment:
```suggestion
Geeft de mogelijkheid om een waarde uit een sleutel-waarde variabele te halen.
```

Reply:
oups, modified this too fast :see_no_evil: